internal/services/oracle/oracle_network_anchor_resource.go

```diff
@@ -0,0 +1,392 @@
+}
+
+type DnsForwardingRuleModel struct {
+	DomainNames         string `tfschema:"domain_names"`
+	ForwardingIPAddress string `tfschema:"forwarding_ip_address"`
+}
+
+func (NetworkAnchorResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+			ValidateFunc: validation.All(
+				validation.StringLenBetween(3, 24),
+				validation.StringMatch(regexp.MustCompile(`^[a-zA-Z0-9-]+$`), "Name may include letters, numbers, or hyphens only"),
+			),
+		},
+
+		"resource_group_name": commonschema.ResourceGroupName(),
+
+		"location": commonschema.Location(),
+
+		"resource_anchor_id": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: resourceanchors.ValidateResourceAnchorID,
+		},
+
+		"subnet_id": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: commonids.ValidateSubnetID,
+		},
+
+		"oci_vcn_dns_label": {
+			Type: pluginsdk.TypeString,
+			// O+C if the value not specified, this gets set to the name of the Network Anchor
+			Optional:    true,
+			Computed:    true,
+			Description: "If the `oci_vcn_dns_label` is omitted it will be set to the `name` field of the network anchor resource.",
+			ForceNew:    true,
+		},
+
+		"oci_backup_cidr_block": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			ValidateFunc: validation.IsCIDR,
+		},
+
+		"oracle_dns_forwarding_endpoint_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Optional: true,
+			Default:  false,
+		},
+
+		"oracle_dns_listening_endpoint_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Optional: true,
+			Default:  false,
+		},
+
+		"oracle_to_azure_dns_zone_sync_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Optional: true,
+			Default:  false,
+		},
+
+		"dns_forwarding_rule": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			ForceNew: true,
+			DiffSuppressFunc: func(k, old, new string, d *pluginsdk.ResourceData) bool {
+				// Always suppress diff if we’re reading existing resource, dns_forwarding_rule is not returning
+				if d.Id() != "" {
+					return true
+				}
+				return false
+			},
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"domain_names": {
+						Type:         pluginsdk.TypeString,
+						Required:     true,
+						ValidateFunc: validate.DomainNames,
+					},
+					"forwarding_ip_address": {
+						Type:         pluginsdk.TypeString,
+						Required:     true,
+						ValidateFunc: validation.IsIPAddress,
+					},
+				},
+			},
+		},
+
+		"dns_listening_endpoint_allowed_cidrs": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Computed:     true,
+			ForceNew:     true,
+			ValidateFunc: validate.IsCommaSeparatedCIDRs,
+		},
+
+		"tags":  commonschema.Tags(),
+		"zones": commonschema.ZonesMultipleRequiredForceNew(),
```

Comment:
Let's move tags to the bottom of the schema arguments:
```suggestion
		"zones": commonschema.ZonesMultipleRequiredForceNew(),
		"tags":  commonschema.Tags(),
```